server/constants.py

```diff
@@ -4,7 +4,8 @@
 GRADER_ROLE = 'grader'
 STAFF_ROLE = 'staff'
 INSTRUCTOR_ROLE = 'instructor'
-VALID_ROLES = [STUDENT_ROLE, GRADER_ROLE, STAFF_ROLE, INSTRUCTOR_ROLE]
+LAB_ASSISTANT_ROLE = 'lab_assistant'
+VALID_ROLES = [STUDENT_ROLE, GRADER_ROLE, STAFF_ROLE, INSTRUCTOR_ROLE, LAB_ASSISTANT_ROLE]
```

Comment:
These seem to be in order of increasing permissions, so `LAB_ASSISTANT_ROLE` should be second